app/views/after_sign_up/show.html.erb

```diff
@@ -43,26 +43,28 @@
             <%= link_to "Add a review", new_meeting_review_path(meeting) %>
```

Comment:
This button should go on the dashboard.
Logic:
Button appears when the meeting has happened already. 
The review form redirects to the mentor show